tests/components/zwave/test_init.py

```diff
@@ -83,6 +83,18 @@ def side_effect():
     assert network.options.config_path == 'mock_config_path'
 
 
+async def test_erronous_network_key(hass, mock_openzwave):
+    """Test erronous network key."""
+    result = await async_setup_component(hass, 'zwave', {
```

Comment:
This is unit test, you don't need setup component, just test config validation is enough. However you should test more negative scenario and at least one positive scenario.

You can do something like
```python
with pytest.raises(vol.Invalid):
    result = zwave.CONFIG_SCHEMA({'zwave': {'network_key': 'invalid'}})
```

Forgive me for the typo, I keyed in on my phone.